src/domain/auth/auth.repository.interface.ts

```diff
@@ -21,7 +22,7 @@ export interface IAuthRepository {
 }
 
 @Module({
-  imports: [JwtModule],
+  imports: [JwtModule, SafeRepositoryModule],
```

Comment:
The `SafeRepositoryModule` shouldn't be required.

Reply:
Removed in 72f78b8985d92a7279ab99aafc3de01e84ecfdd7.